png.h

```diff
@@ -3346,7 +3346,7 @@ PNG_EXPORT(244, int, png_set_option, (png_structrp png_ptr, int option,
  * one to use is one more than this.)
  */
 #ifdef PNG_EXPORT_LAST_ORDINAL
-  PNG_EXPORT_LAST_ORDINAL(251);
+  PNG_EXPORT_LAST_ORDINAL(259);
```

Comment:
So we'll git-merge all this into the libpng18 branch as planned, but beyond that, these ordinals gotta go.

Reply:
```suggestion
  PNG_EXPORT_LAST_ORDINAL(259);
```
At least the script (only in the configure build) that checks the ordinals then faults the build unless the machine-generated files have been updated too.

1.8; a simple change that is unlikely to cause merge conflicts against 1.6 since I believe it's only a few lines.  Remove the symbol-by-ordinal support.  Throw that PoS against the wall and see what breaks.  But don't make any other change to those damned macros because if it has to be reintroduced the result would be a nightmare bug farm.

Completely off topic: the same applies to pnglibconf.h.prebuilt, but generate that in the tarball for the "approved" Linux config.  Just don't check it in autoconf or cmake builds that use it.

I think you said this years ago, but if it wasn't you it wasn't me or Glenn either yet it is still correct: The machine generated files should not be in the Git.